adoptedStyleSheets.js

```diff
@@ -128,11 +168,9 @@
         });
         this._adopted = uniqueSheets;
         
-        if (this.isConnected) {
-          sheets.forEach(sheet => {
-            appendContent(this, sheet);
-          });
-        }
+        sheets.forEach(sheet => {
```

Comment:
Any reason you removed `this.isConnected`?

Reply:
Yes, but I'm not sure why I do that 2 weeks ago. I think it's should be, If we are checking is connected, the nodes that not connected will lost outside modifies.

Reply:
I _think_ this might throw on Chrome, make sure you check there, Firefox and Sarari. If all looks good, I'm OK with this.